_sass/components/_button.scss

```diff
@@ -17,7 +17,7 @@
 		cursor: pointer;
 		display: inline-block;
 		font-size: 1em;
-		font-weight: _font(weight-heading);
+		font-weight: _font(weight-bold);
 		height: 2.5em;
 		letter-spacing: _font(kerning-heading);
```

Comment:
`0.03em` is not the right letter-spacing we should be using, it should be not declared for `button`.